addons/stock/models/product.py

```diff
@@ -636,17 +636,6 @@ def _filter_to_unlink(self):
     def _count_returned_sn_products(self, sn_lot):
         return 0
 
-    def filter_has_routes(self):
-        """ Return products with route_ids
-            or whose categ_id has total_route_ids.
-        """
-        products_with_routes = self.env['product.product']
-        # retrieve products with route_ids
-        products_with_routes += self.search([('id', 'in', self.ids), ('route_ids', '!=', False)])
-        # retrive products with categ_ids having routes
-        products_with_routes += self.search([('id', 'in', (self - products_with_routes).ids), ('categ_id.total_route_ids', '!=', False)])
-        return products_with_routes
-
```

Comment:
Don't delete a method in stable (we don't know if a custom module use it), we usually do this kind of clean in the forward port in master

Reply:
Will patch the last fw port to master. Restored it in this PR for now